server/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -622,6 +622,12 @@ public MemoryUsage overLimit(MemoryUsage memoryUsed) {
                         long now = timeSupplier.getAsLong();
                         this.lastCheckTime = now;
                         allocationDuration = now - begin;
+                    } else {
+                        logger.info(
+                            "not attempting to trigger G1GC due to time interval [{}] being lower than [{}]",
+                            begin - lastCheckTime,
+                            minimumInterval
+                        );
                     }
                 }
```

Comment:
Should we also add a log line when acquiring the lock fails?

Reply:
yes, I think so. What will happen if the acquiring the lock fails, a CBE right? In which case, a log line would be helpful.

Reply:
I aggree, added